File: exam-slammr-webapp/src/main/java/uk/co/claritysoftware/exam/slammr/webapp/persistence/dynamodb/repository/DynamoDbQuestionItemRepository.java
package uk.co.claritysoftware.exam.slammr.webapp.persistence.dynamodb.repository;

import org.socialsignin.spring.data.dynamodb.repository.EnableScan;
import org.springframework.data.repository.CrudRepository;
import uk.co.claritysoftware.exam.slammr.webapp.persistence.dynamodb.item.question.QuestionItem;

/**
 * Spring Data Spring Social repository for {@link QuestionItem}
 */
@EnableScan
public interface DynamoDbQuestionItemRepository extends CrudRepository<QuestionItem, String> {

}

File: exam-slammr-webapp/src/main/kotlin/uk/co/claritysoftware/exam/slammr/webapp/persistence/dynamodb/repository/DynamoDbQuestionItemRepository.kt
package uk.co.claritysoftware.exam.slammr.webapp.persistence.dynamodb.repository

import org.socialsignin.spring.data.dynamodb.repository.EnableScan
import org.springframework.data.repository.CrudRepository
import uk.co.claritysoftware.exam.slammr.webapp.persistence.dynamodb.item.question.QuestionItem

/**
 * Spring Data Spring Social repository for [QuestionItem]
 */
@EnableScan
interface DynamoDbQuestionItemRepository : CrudRepository<QuestionItem, String>
